README.md

```diff
@@ -56,11 +56,11 @@ Main features of diktat are the following:
 curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.43.2/ktlint && chmod a+x ktlint
```

Comment:
do we need to update `ktlikt` version here?

Reply:
Yeah, we do. We should also add it to our automation